xdsl/interpreter.py

```diff
@@ -425,7 +427,16 @@ def will_interpret_op(self, op: Operation, args: PythonValues) -> None:
         def did_interpret_op(self, op: Operation, results: PythonValues) -> None:
             ...
 
+    SYSTEM_BITWIDTH: ClassVar[int | None] = {"64bit": 64, "32bit": 32}.get(
+        platform.architecture()[0]
+    )
```

Comment:
I think you meant to add a default value of `None` to the `.get` here, as referenced in the next line.

Reply:
Oooooh, damn, didn't know that!